### ISSUE TITLE
Exception java.lang.ArrayIndexOutOfBoundsException - not informative

### ISSUE THREAD

**Issue by zahimizrahi** [TechnionYP5777/Bugquery#184]
Hi BugQuery,

Your application is awesome, by the way :+1:

I tried to do the following example project as described in my wiki https://github.com/TechnionYP5777/Bugquery/wiki/Experience-In-Installation---Zahi-Mizrahi

but I got those results: 

![image](https://user-images.githubusercontent.com/23057130/27769079-5aea92f6-5f2a-11e7-94ac-a8dd33918f21.png)

It's not showing me results of this type of exception, but NullPointerExceptions.

It's very basic exception of trying to access to something to out of the bounds of the array, but the results I got doesn't help me at all 😕

**Comment by yonzarecki** [comment]
It looks like we detected your first line as a trace. 
We usually work with the first exception we find in the input, as expected in regular inputs.
Thank you for noting this to us. :smiley:

**Comment by zahimizrahi** [comment]
It should be worked with printings, I guess... It must ignore the prints and take into consideration the exception only.

**Comment by yosefraisman** [comment]
We looked into this issue at the beginning of the semester, and found out that extracting a specific type of output from a console is actually a pain, and sometimes makes us lose exceptions that were printed regularly and not into the error stream.

It's also a costly venture, extracting the correct trace, in terms of developer time - we did our best, and thought that in the future whoever inherits the project could improve on the algorithm. I personally think implementing a launch configuration mechanism will solve this particular problem.

Thanks again for your input.
